specs/driver_spec.rb

```diff
@@ -76,5 +83,99 @@
       driver = RideShare::Driver.new(id: 54, name: "Rogers Bartell IV", vin: "1C9EVBRM0YBC564DZ")
       driver.average_rating.must_equal 0
     end
-  end
-end
+  end # end of describe "average_rating method"
+
+  describe "total_revenue method" do
+    it "calculate the total revenue" do
+      trips = [
```

Comment:
One case I don't see your tests covering, for all the wave 1 methods, is what happens for a driver or passenger that has no trips at all?